app/scripts/directives/mobileAssociations.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+(function() {
```

Comment:
I would like to understand why some controllers are created inside functions like this, and some are not, do you know what the impact is?

Reply:
This IIFE pattern was used to avoid leaking variables and functions into global space and getting clashes. Modern bundling handles this now so it's not needed. (I just grabbed a component skeleton to begin this one so I have it)